detection_rules/main.py

```diff
@@ -29,8 +24,15 @@
 
 
 @click.group('detection-rules', context_settings={'help_option_names': ['-h', '--help']})
-def root():
+@click.option('--debug/--no-debug', '-d/-n', is_flag=True, default=None,
+              help='Print full exception stacktrace on errors')
```

Comment:
the `--no-debug` flag is probably redundant.
also `-D` would be less likely to collide with flags from other commands, since it's capital. which also gives an impression that it's global. thoughts on this?
```suggestion
@click.option('--debug', '-D', is_flag=True, default=None, help='Print full exception stacktrace on errors')
```

Reply:
Main reason for the `--no-debug` (from the CLI.md):

>Precedence goes to the flag over the config file, so if debug is enabled in your config and you run 
`python -m detection-rules --no-debug`, debugging will be disabled.

I do like the idea of `-D` to infer globalness though